wallet/src/wallet/mod.rs

```diff
@@ -222,6 +300,8 @@ impl<B: storage::Backend> Wallet<B> {
             accounts,
             latest_median_time,
             unsynced_accounts,
+            state: WalletState::Syncing,
+            next_unused_account,
```

Comment:
Can you please explain why we need to track this? What has changed in this PR that necessitates it?